src/types/index.ts

```diff
@@ -87,6 +87,24 @@ export interface IAccountLabeled extends Partial<IAccount>{
   label?: TranslateResult,
 }
 
+export interface IMultisig {
```

Comment:
`IMultisig` in my opinion looks too generic. I suggest to rename this to `IMultisigVault` or `IMultisigAccount`...